F_NODE_SAMPLE_FREQUENCY: 2.0
I_SCREEN_OFFSET_X: 0
I_SCREEN_OFFSET_Y: 0
I_SCREEN_RESOLUTION_X: 1280
I_SCREEN_RESOLUTION_Y: 720
F_SCREEN_RESOLUTION_SCALE: 0.75
